bundle/android/strings.xml

```diff
@@ -6,7 +6,7 @@
     <string name="gutenberg_native_1_s_2_s_is_3_s_4_s" tools:ignore="UnusedResources">%1$s. %2$s is %3$s %4$s.</string>
     <!-- translators: 1: From block title, e.g. Paragraph. 2: To block title, e.g. Header. -->
     <string name="gutenberg_native_1_s_transformed_to_2_s" tools:ignore="UnusedResources">%1$s transformed to %2$s</string>
-    <string name="gutenberg_native_add_a_new_block_at_any_time_by_tapping_on_the_icon_in_the_toolbar" tools:ignore="UnusedResources">Add a new block at any time by tapping on the + icon in the toolbar on the bottom left. </string>
```

Comment:
@dcalhoun, this seems like an erroneous change that might be resolved by updating and re-running the bundle? I'll go ahead to approve regardless, but just wanted to make a note in case it slipped through.

Reply:
@SiobhyB I agree the inclusion of the change is odd. 

Based on the diff (see [line directly beneath this one](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3930/files#diff-d116ecb4f40d4fa795c1a136e7fddfef1850a015725f83835d7d9ee33f04ee64R9)), it would appear a single space character (` `) is removed from the end of this preexisting translation string. So, I am not sure as to the origin of the change, but I cannot imagine the change is harmful. If anything it appears more accurate to me. 

My best guess is that the change relates to recent i18n tooling improvements @fluiddot completed. Maybe some of the changes now parse the [relevant string](https://github.com/WordPress/gutenberg/blob/5a84b27bcead1528ca98bcade9c6d5259c70de7b/packages/editor/src/components/editor-help/add-blocks.native.js#L27) differently/better.

Reply:
Ah, I see, interesting. I know there was recently talk of how whitespace can cause issues for translations in https://github.com/WordPress/gutenberg/pull/37956, so it'd make sense if there was now an automated way to remove any whitespace. It'd be interesting to get confirmation around that, but definitely agree it's not a blocker for this PR or a change that appears like it'd be harmful in anyway.

Reply:
Right, sorry for that, I should have updated the bundle and localization strings files in [this PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4496) for preventing this case, thanks for catching it 🙇 .